helpers/build-gmt-master.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Download and build the GMT master branch. Used by TravisCI.
+
+set -e
+
+mkdir -p "$INSTALLDIR"
+mkdir -p "$COASTLINEDIR"
+git clone --depth=1 --branch=master https://github.com/GenericMappingTools/gmt.git tmp
```

Comment:
Why not clone it to gmt-master directly? Any side-effects?

Reply:
It's because of the cache. I'm having Travis cache `gmt-master/build` so we don't recompile things that didn't change. But git won't checkout into an existing directory.

Reply:
Got it.